### PR TITLE
Use protobuf between agent/analyzer as first level protocol

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

**Comment by safchain** [comment]
run functional-tests-backend-orientdb

File: analyzer/flow_server.go [hunk 1]
```diff
@@ -101,11 +101,15 @@ type FlowServer struct {
 
 // OnMessage event
 func (c *FlowServerWebSocketConn) OnMessage(client ws.Speaker, m ws.Message) {
-	f, err := flow.FromData(m.Bytes(client.GetClientProtocol()))
-	if err != nil {
+	// rawmessage at this point
+	b, _ := m.Bytes("")
```
Comment (lebauce) [inline analyzer/flow_server.go]: nit: using a constant would probably be clearer

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
run skydive-cdd-overview-tests

**Comment by lebauce** [comment]
run skydive-python-tests

**Comment by safchain** [comment]
rerun skydive-cdd-overview-tests

**Comment by safchain** [comment]
rerun skydive-functional-tests-backend-elasticsearch

**Pull request merged**